<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Commands</class>
 <widget class="QDialog" name="Commands">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="A">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Q">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>Q</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="AQ_explanation">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="Z">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="S">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="SZ_explanation">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="E">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>E</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="D">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ED_explanation">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="Key4">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Key7">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Key47_explanation">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="Key5">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Key8">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Key58_explanation">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="Key6">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Key9">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Key69_explanation">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="Minus">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Plus">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MinusPlus_explanation">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
